R/db-odbc-mssql.R

```diff
@@ -100,3 +100,41 @@
   )
   DBI::dbExecute(con, sql)
 }
+
+#' @export
+`db_compute.Microsoft SQL Server` <- function(con, table, sql, temporary=TRUE, unique_indexes=list(), indexes=list(), ...)
+{
+  # check that name has prefixed '##' if temporary
+  if(temporary && substr(table, 1, 1) != "#")
```

Comment:
I think it would be better to modify `db_compute.DBIConnection` to have:

```R
table <- db_save_query(con, sql, table, temporary = temporary)
```

Reply:
Hm, could this affect other databases?